docs/content/users/topics/sharing.md

```diff
@@ -20,8 +20,8 @@ There are CMSes that make this a little harder, especially WordPress and Magento
 
 ### Setting up a Stable ngrok Subdomain
 
-1. Get a paid token with at least the basic plan, and configure it. It will be in `~/.ngrok2/ngrok.yml` as `authtoken`.
-2. Configure `ngrok_args` to use a stable subdomain. In `.ddev/config.yaml`, `ngrok_args: --subdomain wp23` will result in ngrok always using `wp23.ngrok.io` as the URL, so it’s not changing on you all the time.
+1. Get a token with at least the free plan from [ngrok.com](https://ngrok.com/), and configure it. It will be in `~/.ngrok2/ngrok.yml` as `authtoken`.
```

Comment:
Can you do a stable domain with free plan now? You didn't used to be able to do that.

Reply:
Stable is only on paid plan.

Reply:
So the original verbiage was correct in that case right? You have to get a token on a paid plan. Not free plan as this PR suggests.

Reply:
I read it as "Get a token (any plan will do) and to configure it"  as a first step to setting up NGROK, 
Then, for the stable subdomain do .... but this requires a paid plan.

I find the original wording awkward.  @mattstein probably has the missing elegance.

Reply:
You only need a paid plan if you want to use the "stable domain feature".

Reply:
you're right. just decline the PR then?